### PR TITLE
add a use flag to disable x86 asm

### PR DESCRIPTION
**Pull request by wolfwood**
while its not actually necessary in my case, this may be of use to someone else.  will only build with USE="-asm" for a version incorporating https://github.com/MaartenBaert/ssr/commit/8043d49b428598c96b9903c8281d8a26fab8c196

cheers.

### CODE REVIEW

**Comment by wolfwood** [comment]
actually I guess that means I should edit version 9999 not the current version.  will fix.

**Comment by anders-larsson** [comment]
Isn't this issue fixed by the commit you mentioned or are we still required to send --disable-asm for it to take that code path?

**Comment by wolfwood** [comment]
the commit to SSR fixed building with './configure --disable-asm' so that it sucessfully completes.  this patch allows someone to specify USE=-asm so that the package configures with --disable-asm.

**Comment by anders-larsson** [comment]
Ok. Would you mind rebasing ths on the latest HEAD (just did a version bump) and it's currently not mergeable. You might want to modify my version bump (0.3.3) as well (just released). You might want to set a correct author name/mail as well!

**Comment by anders-larsson** [comment]
Sorry. This commit was added _after_ version 0.3.3 was released. It should only be added to 9999.

**Comment by wolfwood** [comment]
should be sorted.

File: media-video/simplescreenrecorder/simplescreenrecorder-9999.ebuild [hunk 1]
```diff
@@ -94,6 +94,7 @@ multilib_src_configure() {
 		$(multilib_native_use_with pulseaudio)
 		$(multilib_native_use_with jack)
 		$(multilib_native_use_with qt5)
+		$(use_enable asm x86_asm)
```
Comment (anders-larsson) [inline media-video/simplescreenrecorder/simplescreenrecorder-9999.ebuild]: Currently it attempts to use ./configure .... --disable-x86_asm. From ./configure --help:
````
  --disable-x86-asm       don't use x86/x64 assembly instructions
````

**Comment by wolfwood** [comment]
both work, but I've switched to the documented flag.

**Comment by anders-larsson** [comment]
Sorry didn't know. I didn't have any good way to verify it's working or not.

Thank you for your contribution!

**Pull request merged**